d3-hexbin/index.d.ts

```diff
@@ -0,0 +1,138 @@
+// Type definitions for D3JS d3-hexbin module v0.2.1
+// Project: https://github.com/d3/d3-hexbin/
+// Definitions by: UNCOVER TRUTH Inc. <https://github.com/uncovertruth/>, Tom Wanzek <https://github.com/tomwanzek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface HexbinBin<T> extends Array<T> {
+    x: number;
+    y: number;
+}
+
+export interface Hexbin<T> {
+    /**
+     * Bins the specified array of points, returning an array of hexagonal bins.
+     * For each point in the specified points array, the x- and y-accessors are
+     * invoked to compute the x- and y-coordinates of the point, which is then
+     * used to determine which hexagonal bin to add the point.
+     * If either the x- or y-coordinate is NaN, the point is ignored and will
+     * not be in any of the returned bins.
+     */
+    (points: T[]): HexbinBin<T>[];
+
+    /**
+     * Returns the SVG path string for the hexagon centered at the origin ⟨0,0⟩.
+     * The path string is defined with relative coordinates such that you can
+     * easily translate the hexagon to the desired position.
+     * If radius is not specified, the hexbin’s current radius is used.
+     * If radius is specified, a hexagon with the specified radius is returned;
+     * this is useful for area-encoded bivariate hexbins.
+     *
+     * @param {number} radius Radius number
```

Comment:
OK, remove `{number}`.
`@param radis An optional radius parameter to be used when generating the hexagon`
is ok?